pom.xml

```diff
@@ -148,6 +148,16 @@
             <scope>test</scope>
             <version>20180130</version>
         </dependency>
+        <dependency>
+            <groupId>fish.payara.extras</groupId>
+            <artifactId>payara-embedded-all</artifactId>
```

Comment:
for SDK, why do we need Payara dependency?

Reply:
We are providing upload functionality in SDK by using FormDataMultiPart which requires this dependency. FormDataMultiPart in hw2 was coming from this jar file, so I did the same thing here.

Reply:
HW2 is a server app so needs this, we don't need this here for a client. Payara dependency adds a lot of dependencies and since this is SDK, adding a bleeding edge version of payara app server is not ideal. We have not been able to upgrade to payara 5.20.0 itself. Please remove and change to something else.
P.S I just happened to see this.

Reply:
Sure Anshu. Thanks for the comment. I will change to some other dependency which brings in FormDataMultiPart.

Reply:
Added         
<dependency>
            <groupId>com.sun.jersey.contribs</groupId>
            <artifactId>jersey-multipart</artifactId>
            <version>1.19.4</version>
</dependency>
for the FormDataMultiPart class.

Reply:
Anshu, Thanks for your input and comments.